proto/src/determined/trial/v1/trial.proto

```diff
@@ -103,3 +103,50 @@ message Trial {
   // Best checkpoint.
   CheckpointWorkload best_checkpoint = 10;
 }
+
+// TrialProfilerMetricLabels are the labels for a single series, where a series
+// is a defined as all metrics sharing a distinct set of labels
+message TrialProfilerMetricLabels {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "trial_id", "name" ] }
+  };
+  // To distinguish the 2 different categories of metrics.
+  enum ProfilerMetricType {
+    // Zero-value (not allowed).
+    PROFILER_METRIC_TYPE_UNSPECIFIED = 0;
+    // For systems metrics, like GPU utilization or memory.
+    PROFILER_METRIC_TYPE_SYSTEM = 1;
+    // For timing metrics, like how long a backwards pass or getting a batch
+    // from the dataloader took.
+    PROFILER_METRIC_TYPE_TIMING = 2;
+  }
+  // The ID of the trial.
+  int32 trial_id = 1;
+  // The name of the metric.
+  string name = 2;
+  // The agent ID associated with the metric.
+  string agent_id = 3;
+  // The GPU UUID associated with the metric.
+  string gpu_uuid = 4;
+  // The type of the metric.
+  ProfilerMetricType metric_type = 5;
```

Comment:
What do you think about using a bool instead of an enum here? Protobuf enums kind of suck from a UX perspective

Reply:
Not really a fan of loading meaning other than true/false into a bool.